src/multidim/intervalbox.jl

```diff
@@ -73,12 +73,12 @@ diam(X::IntervalBox) = maximum(diam.(X.v))
 
 emptyinterval(X::IntervalBox{N,T}) where {N,T} = IntervalBox(emptyinterval.(X.v))
 
-isinf(X::IntervalBox) = any(isinf(X))
+isinf(X::IntervalBox) = any(isinf.(X))
 
 isinterior(X::IntervalBox{N,T}, Y::IntervalBox{N,T}) where {N,T} = all(isinterior.(X, Y))
 
-contains_zero(X::SVector) = all(contains_zero(X))
-contains_zero(X::IntervalBox) = all(contains_zero(X))
+contains_zero(X::SVector) = all(contains_zero.(X))
```

Comment:
Same here

Reply:
Why did you remove the definition for `IntervalBox`?

Reply:
By mistake... sorry. Can you fix that? Or shall I?